starlette/__init__.py

```diff
@@ -1 +1,31 @@
 __version__ = "0.38.2"
+
+from .applications import Starlette
+from .middleware import Middleware
+from .responses import (
+    HTMLResponse,
+    JSONResponse,
+    PlainTextResponse,
+    RedirectResponse,
+    Response,
+    StreamingResponse,
+)
+from .routing import Mount, Route, Router, WebSocketRoute
+from .websockets import WebSocket
+
+__all__ = (
+    "__version__",
+    "Starlette",
+    "Middleware",
+    "Response",
+    "HTMLResponse",
+    "JSONResponse",
+    "PlainTextResponse",
+    "RedirectResponse",
+    "StreamingResponse",
+    "Route",
+    "Router",
+    "Mount",
+    "WebSocketRoute",
+    "WebSocket",
+)
```

Comment:
Reviewing the complete set of documented imports here's what I've got...

```python
from starlette.applications import Starlette
from starlette.authentication import requires, AuthCredentials, AuthenticationBackend, AuthenticationError, SimpleUser
from starlette.background import BackgroundTask
from starlette.config import environ, Config
from starlette.datastructures import Headers, MutableHeaders, Secret, URL
from starlette.endpoints import HTTPEndpoint
from starlette.exceptions import HTTPException, WebSocketException
from starlette.middleware import Middleware
from starlette.middleware.base import BaseHTTPMiddleware
from starlette.middleware.cors import CORSMiddleware
from starlette.middleware.gzip import GZipMiddleware
from starlette.middleware.httpsredirect import HTTPSRedirectMiddleware
from starlette.middleware.sessions import SessionMiddleware
from starlette.middleware.trustedhost import TrustedHostMiddleware
from starlette.responses import Response, FileResponse, HTMLResponse, JSONResponse, PlainTextResponse, RedirectResponse, StreamingResponse
from starlette.requests import Request
from starlette.routing import Route, Mount, WebSocketRoute
from starlette.schemas import SchemaGenerator
from starlette.staticfiles import StaticFiles
from starlette.templating import Jinja2Templates
from starlette.testclient import TestClient
from starlette.websockets import WebSocket
```

Does it make sense to include the whole set of imports here, or just the subset above?